darts/models/forecasting/forecasting_model.py

```diff
@@ -440,20 +440,19 @@ def _get_historical_forecastable_time_index(
             max_future_cov_lag,
         ) = self.extreme_lags
 
-        intersect_ = None
+        if min_target_lag is None:
+            min_target_lag = 0
 
-        # target longest possible time index
-        if (min_target_lag is not None) and (series is not None):
-            intersect_ = generate_index(
-                start=series.start_time()
-                - (min_target_lag - max_target_lag) * series.freq
-                if is_training
-                else series.start_time() - min_target_lag * series.freq,
-                end=series.end_time(),
-                freq=series.freq,
-            )
+        # longest possible time index for target
```

Comment:
Thanks :) Could you add some unit tests which check the issue?
It's always good when fixing a bug to add a unit test, because we apparently haven't checked it before